src/core/xtc.ml

```diff
@@ -1,6 +1,7 @@
 (** This module provides a function to translate a signature to the XTC format
     used in the termination competition.
-    @see <http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB/file/tip/xml/xtc.xsd>
+    @see:
+    <http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB/file/tip/xml/xtc.xsd>
```

Comment:
I think @fblanqui removed the line break on purpose for ocamldoc formatting.

Reply:
It's for sanity check.

Reply:
Yes, you are right @gabrielhdt ;-)

Reply:
OK it's just because the line is too long.